includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1302,22 +1306,22 @@ public function get_intent_from_order( $order ) {
 	/**
 	 * Retrieves intent from Stripe API by intent id.
 	 *
-	 * @param string $intent_type 	Either 'payment_intents' or 'setup_intents'.
-	 * @param string $intent_id		Intent id.
-	 * @return object|bool 			Either the intent object or `false`.
-	 * @throws Exception 			Throws exception for unknown $intent_type.
+	 * @param string $intent_type   Either 'payment_intents' or 'setup_intents'.
+	 * @param string $intent_id     Intent id.
+	 * @return object|bool          Either the intent object or `false`.
+	 * @throws Exception            Throws exception for unknown $intent_type.
 	 */
 	private function get_intent( $intent_type, $intent_id ) {
 		if ( ! in_array( $intent_type, [ 'payment_intents', 'setup_intents' ] ) ) {
 			throw new Exception( "Failed to get intent of type $intent_type. Type is not allowed" );
 		}
 
-		$response = WC_Stripe_API::request( array(), "$intent_type/$intent_id", 'GET' );
+		$response = WC_Stripe_API::request( [], "$intent_type/$intent_id", 'GET' );
 
 		if ( $response && isset( $response->{ 'error' } ) ) {
 			$error_response_message = print_r( $response, true );
```

Comment:
We have some warnings on `print_r`, do we need to fix those or are we fixing only the errors in this PR?